src/get-own-property-symbols.js

```diff
@@ -21,8 +21,9 @@
     gOPD = Object[GOPD],
     create = Object.create,
     keys = Object.keys,
+    freeze = Object.freeze || Object,
     defineProperty = Object[DP],
-    defineProperties = Object[DPies],
+    $defineProperties = Object[DPies],
```

Comment:
it's mandatory if you want to preserve the function name like I do here
https://github.com/es-shims/get-own-property-symbols/blob/master/build/get-own-property-symbols.max.js#L161-L173